packages/astro/src/content/template/virtual-mod.mjs

```diff
@@ -3,28 +3,33 @@ import {
 	createCollectionToGlobResultMap,
 	createGetCollection,
 	createGetEntryBySlug,
+	createGetEntry,
+	createGetEntries,
+	createGetDataEntryById,
+	createReference,
 } from 'astro/content/runtime';
 
 export { z } from 'astro/zod';
 
-export function defineCollection(config) {
-	return config;
-}
-
-// TODO: Remove this when having this fallback is no longer relevant. 2.3? 3.0? - erika, 2023-04-04
-export const image = () => {
-	throw new Error(
-		'Importing `image()` from `astro:content` is no longer supported. See https://docs.astro.build/en/guides/assets/#update-content-collections-schemas for our new import instructions.'
-	);
-};
-
 const contentDir = '@@CONTENT_DIR@@';
 
-const entryGlob = import.meta.glob('@@ENTRY_GLOB_PATH@@', {
-	query: { astroContent: true },
+const contentEntryGlob = import.meta.glob('@@CONTENT_ENTRY_GLOB_PATH@@', {
```

Comment:
We should have these unique strings saved somewhere, all together. Or at least follow one pattern for consistency: `'@@CONTENT_DIR@@'` is held in a variable; instead, this one is not.

Reply:
Ah, we sadly can't change this for glob paths. Vite requires all glob imports to be static strings, and **does not support variables.** Messy, but it's the only way to use regex replace 🤷